### DISCUSSION TITLE
Why nullable JSON type complain when value is null?

### DISCUSSION THREAD

**Discussion by fkaanoz** [prisma/prisma#14894] [Q&A]
Hi, 

In schema.prisma file, I used "Json?" as a type for some fields. 
When I try passing null to these fields, prisma gives me an error and it ask me to use "undefined" instead of "null"? 
"undefined" is working well, but why? 

`Argument qsv for data.qsv must not be null. Please use undefined instead.`

Thank you.

**Comment by ludralph** [comment]
Hi @fkaanoz 👋 

There are two types of null values possible for a JSON field in an SQL database.

Database NULL: The value in the database is a NULL.
JSON null: The value in the database contains a JSON value that is null.

To insert a null value into a Json field, you would write:
```js
import { Prisma } from '@prisma/client'

prisma.log.create({
  data: {
    meta: Prisma.JsonNull,
  },
})
```
Do have a look at the prisma [docs](https://www.prisma.io/docs/concepts/components/prisma-client/working-with-fields/working-with-json-fields#filtering-by-null-values) for more information on working with Json fields.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
